distributed/dashboard/components/scheduler.py

```diff
@@ -377,9 +380,34 @@ def __init__(self, scheduler, width=600, **kwargs):
             self.root.toolbar_location = None
             self.root.yaxis.visible = False
 
-            hover = HoverTool()
-            hover.tooltips = "@worker: @proc_memory{0.00 b} (@width{0.00 b} @memtype)"
-            hover.point_policy = "follow_mouse"
+            hover = HoverTool(
+                point_policy="follow_mouse",
+                tooltips="""
+                            <div>
+                                <span style="font-size: 12px; font-weight: bold;">@worker:</span>&nbsp;
+                            </div>
+                            <div>
+                                <span style="font-size: 12px; font-weight: bold;">Process memory (RSS):</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@proc_memory</span>
+                            </div>
+                            <div style="margin-left: 1.5em;">
+                                <span style="font-size: 12px; font-weight: bold;">In process memory:</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@man_in_mem</span>
+                            </div>
+                            <div style="margin-left: 1.5em;">
+                                <span style="font-size: 12px; font-weight: bold;">Spilled to disk:</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@man_spilled</span>
+                            </div>
+                            <div style="margin-left: 1.5em;">
+                                <span style="font-size: 12px; font-weight: bold;">Unmanaged (old):</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@unman_old</span>
+                            </div>
+                            <div style="margin-left: 1.5em;">
+                                <span style="font-size: 12px; font-weight: bold;">Unmanaged (recent):</span>&nbsp;
+                                <span style="font-size: 10px; font-family: Monaco, monospace;">@unman_rec</span>
+                            </div>
+                            """,
```

Comment:
You could completely remove the style tags from here and put them in the .css files instead